modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/SW360ReleaseClient.java

```diff
@@ -80,46 +87,65 @@ public SW360Release getRelease(String releaseId, HttpHeaders header) {
+            return Collections.emptyList();
+        }
 
         if (response.getStatusCode().is2xxSuccessful()) {
             return getSw360SparseReleases(response);
         } else {
-            throw new ExecutionException("Request to get release with externalId " + url + " failed with "
-                    + response.getStatusCode());
+            LOGGER.debug("Request to get releases with externalId {} failed with {}", url, response.getStatusCode());
+            return Collections.emptyList();
         }
     }
 
     public SW360Release createRelease(SW360Release sw360Release, HttpHeaders header) {
         HttpEntity<String> httpEntity = RestUtils.convertSW360ResourceToHttpEntity(sw360Release, header);
 
-        ResponseEntity<Resource<SW360Release>> response = doRestPOST(getEndpoint(), httpEntity,
-                new ParameterizedTypeReference<Resource<SW360Release>>() {});
+        ResponseEntity<Resource<SW360Release>> response;
+        try {
+            response = doRestPOST(getEndpoint(), httpEntity,
+                    new ParameterizedTypeReference<Resource<SW360Release>>() {});
+        } catch (HttpClientErrorException | HttpServerErrorException e) {
+            LOGGER.error("Request to create release {} failed with {}", sw360Release.getName(), e.getStatusCode());
+            return sw360Release;
+        }
 
         if (response.getStatusCode() == HttpStatus.CREATED) {
             return Optional.ofNullable(response.getBody())
-                    .orElseThrow(() -> new ExecutionException("Body was null"))
+                    .orElseThrow(() -> new ExecutionException("Request to get release " + sw360Release.getName() + " returned empty body"))
                     .getContent();
         } else {
-            throw new ExecutionException("Request to create release " + sw360Release.getName() + " failed with "
-                    + response.getStatusCode());
+            LOGGER.error("Request to create release {} failed with {}", sw360Release.getName(), response.getStatusCode());
+            return sw360Release;
         }
     }
 
     public SW360Release patchRelease(SW360Release sw360Release, HttpHeaders header) {
         HttpEntity<String> httpEntity = RestUtils.convertSW360ResourceToHttpEntity(sw360Release, header);
 
-        ResponseEntity<Resource<SW360Release>> response = doRestPATCH(getEndpoint() + "/" + sw360Release.getReleaseId(), httpEntity,
-                new ParameterizedTypeReference<Resource<SW360Release>>() {});
+        ResponseEntity<Resource<SW360Release>> response;
+        try {
+            response = doRestPATCH(getEndpoint() + "/" + sw360Release.getReleaseId(), httpEntity,
+                    new ParameterizedTypeReference<Resource<SW360Release>>() {});
+        } catch (HttpClientErrorException | HttpServerErrorException e) {
+            LOGGER.error("Request to update release {} failed with {}", sw360Release.getName(), e.getStatusCode());
```

Comment:
Just came to my mind, but this is valid for all other occurences as well. It is always bad to simply swallow the exception completely. Proposal, please add a second logger call with DEBUG level that logs the exception not only the status code.